src/js/jsx/tools/GuidesOverlay.jsx

```diff
@@ -59,6 +59,13 @@ define(function (require, exports, module) {
          */
         _scrimGroup: null,
 
+        /**
+         * Flag to keep track of mounted state
+         *
+         * @type {boolean}
+         */
+        _mounted: null,
```

Comment:
I can't rationally explain why any of these `isMounted` calls would be necessary here to begin with, since all the calls happen either synchronously in the same frame as a React library call or in a window resize handler. Maybe we can just remove these?